.github/renovate.json

```diff
@@ -1,10 +1,10 @@
 {
 	"extends": [ "config:base" ],
-	"labels": [ "[Type] Janitorial", "[Status] Needs Review" ],
+	"labels": [ "[Type] Janitorial" ],
 	"prHourlyLimit": 1,
 	"supportPolicy": [ "lts_latest" ],
 	"timezone": "UTC",
-	"schedule": [ "every weekend" ],
+	"schedule": [ "on the last Wednesday of the month" ],
```

Comment:
Does this work? The docs for renovate say they use later.js, which [doesn't seem to support that](https://jsfiddle.net/r7czf60y/). Unfortunately at first glance there doesn't seem to be a particularly straightforward way to handle "the last Wednesday", the closest I can come is something like
```
"schedule": [
    "on Wednesday on the 25th through 31st day of January, March, May, July, August, October, December",
    "on Wednesday on the 24th through 30th day of April, June, September, November",
    "on Wednesday on the 22nd through 29th day of February"
]
```
(but note that'll run twice whenever Feb 29 falls on a Wednesday, which next happens in 2040).

Also it was originally changed to "every weekend" in #12650, seemingly because it submitting PRs that then needed to run CI was making CI slow for other things. So we might want to do something like "before 8:00 am on Wednesday ...".

Reply:
We could also do "1st of the month at 0300". The key is just wait until after branch cut so we can clean this up and preserve the intent.

Reply:
I switched to a monthly preset supported by Renovate in 31051af8aea3d051c23e6c084ab262bec75118d6. As long as it is triggers once a month in the beginning of the month it should be okay I think.